profile/README.md

```diff
@@ -1,51 +1,38 @@
-# Welcome to PUT Motorsport 
-We are a Formula Student-class racing team based at Poznan University of Technology. 
-The Team was founded in 2014. Since the beginning of our activity, we have constructed 7 cars(five combustion and two electric ones), that have achieved
-numerous successes in international competitions. Currently we are working on our third electric vehicle - PM08.
-## In our repositories you can find all of electronic and software projects done in the electric era
+# Welcome to PUT Motorsport
 
-### PCB oriented articles:
-[Low Voltage BMS](https://github.com/PUT-Motorsport/.github/blob/main/LV_BMS-2022-article.pdf)
+We are Formula Student racing team based at Poznan University of Technology.
 
-[PCB stack-up motherboard](https://github.com/PUT-Motorsport/PUTM_EV_FRONT_BOX_2022/blob/main/FRONT_BOX-2022-Article.pdf)
+The team was founded in 2014. Since the beginning of our activity, we have constructed seven racing cars — five combustion and two electric ones — that have achieved numerous successes in international competitions.
 
-[Brake light](https://github.com/PUT-Motorsport/.github/blob/main/brake-light-article.pdf)
+Currently, we are working on our third electric vehicle: PM08.
 
-[Inverter DC-link capacitors](https://github.com/PUT-Motorsport/.github/blob/main/Inverter%20dc%20link%20capacitors.pdf)
+Follow us on:
+- [Instagram](https://www.instagram.com/putmotorsport)
+- [Facebook](https://www.facebook.com/putmotorsport)
+- [YouTube](https://www.youtube.com/putmotorsport)
+- [TikTok](https://www.tiktok.com/@putmotorsport)
+- [LinkedIn](https://www.linkedin.com/company/putmotorsport)
+- [Our website](https://putmotorsport.pl)
 
-[Accelerator Pedal Position Sensor](https://github.com/PUT-Motorsport/.github/blob/main/APPS_article.pdf)
+## About Formula Student Competition
 
-[Contactless Tire Temperature Sensor](https://github.com/PUT-Motorsport/.github/blob/main/article-TTSv2.0-1.pdf)
+Formula Student is an international engineering competition organized since 1979 by SAE International. Every year, university students from all over the world take part in events held on the most famous racetracks. During static and dynamic competitions, they present their independently designed and built vehicles.
 
-[Traction Control](https://github.com/PUT-Motorsport/.github/blob/main/TRACTION_CONTROL_article.pdf)
+## PCB-oriented Articles
 
+In our repositories, you can find various electronic and software projects done in the electric era.
```

Comment:
software i firmware

Reply:
chyba że w sumie firmware mieści się w electronic projects bo w zasadzie zawsze to łączymy razem xD

Reply:
to napisałbym w nawiasie ...electronic (hardware and firmware)...